### ISSUE TITLE
Feature Request: automerge

### ISSUE THREAD

**Issue by pcorpet** [Reviewable/Reviewable#1100]
This is a feature in GitHub pull requests which enables to mark a PR to be merged by GitHub once all checks have passed, comments are answered, and review is positive.

<img width="889" alt="image" src="https://github.com/Reviewable/Reviewable/assets/7937848/5efae900-b500-49a3-95fe-dc4b07e4c0e8">

Missing parity with GitHub:
* display whether a PR is enabled for automerge in a PR page
* make it possible to change the automerge status from a PR page

Bonus: update the action button to differentiate a merge done by the reviewer and a automerge done by GitHub on the PR author's behalf and triggered by the review completion.

**Comment by pkaminski** [comment]
We have some rudimentary support here:  the Publish button will indicate when auto-merge has been enabled for a PR via an icon on the button, and a longer text in the dropdown (viz #904).  We'll even require you to use a long-click to publish if doing so might trigger a merge!

You're right that there's no way to turn automerge on/off for a PR at this point, though.

> Bonus: update the action button to differentiate a merge done by the reviewer and a automerge done by GitHub on the PR author's behalf and triggered by the review completion.

I don't quite understand which button you're referring to here -- does the functionality mentioned above cover this?

**Comment by pcorpet** [comment]
> I don't quite understand which button you're referring to here -- does the functionality mentioned above cover this?

Indeed. I think it was so implicit in the flow that I didn't see it was related. Thanks.

**Comment by pcorpet** [comment]
Also the button doesn't show the "automerge enabled" status for the author.

**Comment by pkaminski** [comment]
Oh?  Looking at the code this status display isn't conditional on the user role, so something odd must be going on.  Do you have an example where this is happening?

**Comment by pkaminski** [comment]
Ah, indeed -- we only show the automerge status in the Publish button/dropdown, so if that's not visible (as in this case) then there's no way to tell.  Sorry, that'll be fixed once we move the checkbox to a new place and give you control over it.
